lib/class-wp-rest-widgets-controller.php

```diff
@@ -116,7 +92,7 @@ public function register_routes() {
 			),
 			'schema' => array( $this, 'get_public_item_schema' ),
 		) );
-		register_rest_route( $this->namespace, '/' . $this->rest_base .'/types/(?P<type>[\w-]+)', array(
+		register_rest_route( $this->namespace, '/' . $this->rest_base_types . '/(?P<type>[\w-]+)', array(
```

Comment:
I'll defer to you on the proper architecture. I wasn't aware of `context=help` but if that's the right place for this information then let's do it that way.

The only gap this would leave is how to retrieve schema for a single type, rather than retrieving schema for all registered types. Do we need it? I don't know. I'm thinking primarily of third-party editors which might want to auto-construct an editing form for a widget...